src/observability/MultiTransactionObservabilityManager.ts

```diff
@@ -0,0 +1,32 @@
+import type { TransactionObservabilityManager } from './observabilityTypes'
+
+export class MultiTransactionObservabilityManager implements TransactionObservabilityManager {
```

Comment:
can you add documentation for it?

Reply:
On it (here I don't have goku :( )

Reply:
Doc added, let me know if it is not good enough 🙏